core/iwasm/interpreter/wasm_opcode.h

```diff
@@ -675,242 +675,239 @@ typedef enum WASMAtomicEXTOpcode {
+        HANDLE_OPCODE(WASM_OP_F32_DEMOTE_F64),                      /* 0xb6 */ \
+        HANDLE_OPCODE(WASM_OP_F64_CONVERT_S_I32),                   /* 0xb7 */ \
+        HANDLE_OPCODE(WASM_OP_F64_CONVERT_U_I32),                   /* 0xb8 */ \
+        HANDLE_OPCODE(WASM_OP_F64_CONVERT_S_I64),                   /* 0xb9 */ \
+        HANDLE_OPCODE(WASM_OP_F64_CONVERT_U_I64),                   /* 0xba */ \
+        HANDLE_OPCODE(WASM_OP_F64_PROMOTE_F32),                     /* 0xbb */ \
+        HANDLE_OPCODE(WASM_OP_I32_REINTERPRET_F32),                 /* 0xbc */ \
+        HANDLE_OPCODE(WASM_OP_I64_REINTERPRET_F64),                 /* 0xbd */ \
+        HANDLE_OPCODE(WASM_OP_F32_REINTERPRET_I32),                 /* 0xbe */ \
+        HANDLE_OPCODE(WASM_OP_F64_REINTERPRET_I64),                 /* 0xbf */ \
+        HANDLE_OPCODE(WASM_OP_I32_EXTEND8_S),                       /* 0xc0 */ \
+        HANDLE_OPCODE(WASM_OP_I32_EXTEND16_S),                      /* 0xc1 */ \
+        HANDLE_OPCODE(WASM_OP_I64_EXTEND8_S),                       /* 0xc2 */ \
+        HANDLE_OPCODE(WASM_OP_I64_EXTEND16_S),                      /* 0xc3 */ \
+        HANDLE_OPCODE(WASM_OP_I64_EXTEND32_S),                      /* 0xc4 */ \
+        HANDLE_OPCODE(WASM_OP_DROP_64),                             /* 0xc5 */ \
+        HANDLE_OPCODE(WASM_OP_SELECT_64),                           /* 0xc6 */ \
+        HANDLE_OPCODE(EXT_OP_GET_LOCAL_FAST),                       /* 0xc7 */ \
+        HANDLE_OPCODE(EXT_OP_SET_LOCAL_FAST_I64),                   /* 0xc8 */ \
+        HANDLE_OPCODE(EXT_OP_SET_LOCAL_FAST),                       /* 0xc9 */ \
+        HANDLE_OPCODE(EXT_OP_TEE_LOCAL_FAST),                       /* 0xca */ \
+        HANDLE_OPCODE(EXT_OP_TEE_LOCAL_FAST_I64),                   /* 0xcb */ \
+        HANDLE_OPCODE(EXT_OP_COPY_STACK_TOP),                       /* 0xcc */ \
+        HANDLE_OPCODE(EXT_OP_COPY_STACK_TOP_I64),                   /* 0xcd */ \
+        HANDLE_OPCODE(EXT_OP_COPY_STACK_VALUES),                    /* 0xce */ \
+        HANDLE_OPCODE(WASM_OP_IMPDEP),                              /* 0xcf */ \
+        HANDLE_OPCODE(WASM_OP_REF_NULL),                            /* 0xd0 */ \
+        HANDLE_OPCODE(WASM_OP_REF_IS_NULL),                         /* 0xd1 */ \
+        HANDLE_OPCODE(WASM_OP_REF_FUNC),                            /* 0xd2 */ \
+        HANDLE_OPCODE(EXT_OP_BLOCK),                                /* 0xd3 */ \
+        HANDLE_OPCODE(EXT_OP_LOOP),                                 /* 0xd4 */ \
+        HANDLE_OPCODE(EXT_OP_IF),                                   /* 0xd5 */ \
+        HANDLE_OPCODE(EXT_OP_BR_TABLE_CACHE),                       /* 0xd6 */ \
+        [WASM_OP_MISC_PREFIX] = HANDLE_OPCODE(WASM_OP_MISC_PREFIX), /* 0xfc */ \
+        [WASM_OP_ATOMIC_PREFIX] =                                              \
+            HANDLE_OPCODE(WASM_OP_ATOMIC_PREFIX), /* 0xfe */                   \
```

Comment:
Adding the two lines causes the change of many lines, how about defining a macro to avoid that?
```C
#define SET_GOTO_TABLE_ELEM(opcode) \
[opcode] = HANDLE_OPCODE(opcode)
```
And here:
```C
SET_GOTO_TABLE_ELEM(WASM_OP_MISC_PREFIX),
SET_GOTO_TABLE_ELEM(WASM_OP_ATOMIC_PREFIX),
```

Reply:
sure, that looks better